lib/content-services/search/components/search-control.component.ts

```diff
@@ -73,19 +73,28 @@ export class SearchControlComponent implements OnInit, OnDestroy {
     @Output()
     optionClicked: EventEmitter<any> = new EventEmitter();
 
+    @ViewChild('searchInput')
+    searchInput: ElementRef;
+
+
     searchTerm: string = '';
     subscriptAnimationState: string;
 
     private toggleSearch = new Subject<any>();
     private focusSubject = new Subject<FocusEvent>();
 
     constructor(public authService: AuthenticationService,
+                private renderer: Renderer,
```

Comment:
`Renderer` is marked as `deprecated`

Reply:
Renderer2 lacks this feature and it seems they are not planing to bring it back. The main alternatives floating around is with a `Service` or just `searchInput.nativeElement.focus()`.  I guess this is a team decision